vertx-grpc-protoc-plugin2/src/main/resources/grpc-service.mustache

```diff
@@ -87,62 +87,56 @@ public class {{className}} extends {{serviceName}}Service implements Service {
 {{#allMethods}}
     all.add({{methodName}});
 {{/allMethods}}
+{{#transcodingMethods}}
```

Comment:
as said previously, instead of having:

- all methods
- all transcoding methods

we should have

- all methods, if a method declares transcoding meta data then the method is a transcoding method otherwise it is a regular method